akka-actor/src/main/scala/akka/actor/ActorPath.scala

```diff
@@ -28,9 +28,17 @@ object ActorPath {
    * See [[isValidPathElement()]] for a non-throwing version.
    *
    * @param element actor path element to be validated
+   */
+  final def validatePathElement(element: String): Unit = validatePathElement(element, fullPath = null)
+
+  /**
+   * Validates the given actor path element and throws an [[InvalidActorNameException]] if invalid.
+   * See [[isValidPathElement()]] for a non-throwing version.
+   *
+   * @param element actor path element to be validated
    * @param fullPath optional fullPath element that may be included for better error messages; null if not given
    */
-  final def validatePathElement(element: String, fullPath: String = null): Unit = {
+  final def validatePathElement(element: String, fullPath: String): Unit = {
```

Comment:
IMO good news - this is a new method on master (I have added it in https://github.com/akka/akka/commit/c31469dd2930669324512a7750ecc1cb8e414d8f , on 2.3 we only have `isValidPathElement`) and it's on an object - not a trait, so we're safe from the "method added to trait" problem.

I think we might get away with this change in this case.